geowebcache/core/src/test/java/org/geowebcache/blobstore/file/FileBlobStoreComformanceTest.java

```diff
@@ -27,4 +36,103 @@
     public void createTestUnit() throws Exception {
         this.store = new FileBlobStore(temp.getRoot().getAbsolutePath());
     }
+
+    @Test
+    public void testMetadataWithPointInKey() throws Exception {
+        assertThat(store.getLayerMetadata("testLayer", "test.Key"), nullValue());
+        store.putLayerMetadata("testLayer", "test.Key", "testValue");
+        assertThat(store.getLayerMetadata("testLayer", "test.Key"), equalTo("testValue"));
+    }
+
+    @Test
+    public void testConcurrentMetadataWithPointInKey() throws InterruptedException {
```

Comment:
Looks like these tests share a lot of code, could it be possible to create a support method contains the common code?